src/services/TM_API.coffee

```diff
@@ -126,6 +126,13 @@ class TM_API
     postData = { email:email }
     @.$http.post(url, postData).success callback
 
+
+
+  pwd_reset_Token: (username,token,password, callback)=>
+    url      = "/jade/json/passwordReset/"+ username+ "/"+ token
```

Comment:
we have to be careful with REST calls like this one, since this would allow REST injection (for example to access a rest call in the GraphDB which should not be accessible by the user (and there are a couple)